src/Form/Extension/Field/Type/FormTypeFieldExtension.php

```diff
@@ -189,11 +189,25 @@ public function configureOptions(OptionsResolver $resolver)
             'label_render' => true,
             // NEXT_MAJOR: Remove this property and the deprecation message
             'sonata_help' => null,
-        ])
-            ->setDeprecated(
-                'sonata_help',
-                'The "sonata_help" option is deprecated since sonata-project/admin-bundle 3.60, to be removed in 4.0. Use "help" instead.'
-            );
+        ]);
+
+        // BC layer for symfony/options-resolver < 5.1.
+        // NEXT_MAJOR: remove the check and the contents of the `else` condition.
```

Comment:
IMO this comment is not for next major but for when we drop sf 4.4

Reply:
Anyway on next major this whole deprecated thing will be removed because it is deprecated